GsaGH/Helpers/GsaCOM/ComResultsHelper.cs

```diff
@@ -55,12 +56,16 @@ private enum ElementDimension {
 
       foreach (int nodeId in nodes.Keys) {
         var xyzRes = new ConcurrentDictionary<int, GsaResultQuantity>();
-        var ff = new Ratio((double)gsa.Output_Extract(nodeId, 0), RatioUnit.DecimalFraction);
-        var res = new GsaResultQuantity() {
-          X = ff,
-        };
-        xyzRes.TryAdd(0, res);
-        r.XyzResults.TryAdd(nodeId, xyzRes);
+        try {
+          var ff = new Ratio((double)gsa.Output_Extract(nodeId, 0), RatioUnit.DecimalFraction);
+          var res = new GsaResultQuantity() {
+            X = ff,
+          };
+          xyzRes.TryAdd(0, res);
+          r.XyzResults.TryAdd(nodeId, xyzRes);
+        } catch (System.ArgumentException) {
+          // skip
```

Comment:
when does it throw `ArgumentExceptions`? should we inform the user?

Reply:
When it cannot find footfall result values for a particular node. This is an edge-case, where the user has included points in a 2D Member but those points lie withing void cutters or a cut out by section shapes:
![image](https://github.com/arup-group/GSA-Grasshopper/assets/25223248/716a2888-5537-48ba-840a-709d808f73a2)
The current method is returning max footfall result (transient or resonant) and if the node does not have a value we can simply skip it.

Reply:
Maybe worth putting a comment there ;)

Reply:
We are hopefully getting rid of this class soon with [GSA-7392](https://ovearup.atlassian.net/browse/GSA-7392)